db/migrate/20200519141015_create_storage_resources.rb

```diff
@@ -0,0 +1,15 @@
+class CreateStorageResources < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_resources do |t|
+      t.string :name
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
+      t.string :uuid
+      t.bigint :logical_free
+      t.bigint :logical_total
+      t.string :pool_name
+      t.references :storage_system, :type => :bigint, :index => true, :references => :storage_system
```

Comment:
you only need the `:references => :storage_system` if the name of the model doesn't match the name of the column, e.g. the column name is ems_id but the model name is `ExtManagementSystem`, for anything else it infers the reference from the column name so these can just be `t.references :storage_system, :type => :bigint, :index => true`